config/game_options.txt

```diff
@@ -467,6 +467,7 @@ ROUNDSTART_RACES aquatic
 ROUNDSTART_RACES insect
 ROUNDSTART_RACES synthmammal
 ROUNDSTART_RACES synthhuman
+ROUNDSTART_RACES ghoul
```

Comment:
If your species is almost entirely beneficial it shouldn't be a roundstart race

Reply:
To be fair, synths already exist.